### PR TITLE
PRO-1329: Pillar Wallet - Key-based and Receive modal description

### PR DESCRIPTION
**Pull request by yashvyas19**
## Description

- Removed unnecessary deployment status check for key-based wallets
- Removed 'Etherspot account' label and centered ETH logo for Archanova and Keybased wallets

## Motivation and Context

- https://linear.app/pillarproject/issue/PRO-1329/pillar-wallet-key-based-and-receive-modal-description

## How Has This Been Tested?

- Tested on Simulator 14 PRO
- Tested on Android Emulator

## Screenshots (if appropriate):
Simulator Screenshot - iPhone 14 Pro - 2023-08-17 at 18 04 32

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by Yash-Chaubal — APPROVED** [review]
LGTM! I know it's not part of the ticket, but maybe we should still have a title if it's archanova or keybased?

**Comment by yashvyas19** [comment]
> LGTM! I know it's not part of the ticket, but maybe we should still have a title if it's archanova or keybased?

I'll add a title then for those two

File: src/locales/en/common.json [hunk 1]
```diff
@@ -1691,7 +1691,11 @@
     "activationText": "You need to activate your smart wallet first.",
     "activateText": "Activate",
     "notDeployedText": "⚠️ Sending tokens is unavailable. Currently you can only receive tokens in <0>{{chain}}</0> wallet",
-    "etherspotTitle": "Etherspot Account",
+    "accountName": {
+      "etherspot": "Etherspot Account",
+      "keyBased": "Key Based Account",
```
Comment (Yash-Chaubal) [inline src/locales/en/common.json:1696]:
```suggestion
      "keybased": "Keybased Account",
```

**Review by Yash-Chaubal — APPROVED** [review]

File: src/utils/accounts.js [hunk 2]
```diff
@@ -150,6 +150,19 @@ export const getAccountName = (accountType: AccountTypes | TranslatedString): st
...
+    case isKeyBasedAccount(activeAccount):
+      return t('receiveModal.accountName.keyBased');
+    case isArchanovaAccount(activeAccount):
+      return t('receiveModal.accountName.archanova');
+    case isEtherspotAccount(activeAccount):
+      return t('receiveModal.accountName.etherspot');
+    default:
+      return '';
+  }
+};
```
Comment (Yash-Chaubal) [inline src/utils/accounts.js:164]: honestly I think you should just just if statements here

**Pull request merged**